### PR TITLE
2008 czub

### PR DESCRIPTION
**Pull request by fkalholm**
adding 2008 paper

### CODE REVIEW

**Comment by fkalholm** [comment]
energies does not seem to match well. Its always below what is expected.

**Comment by nbassler** [comment]
> energies does not seem to match well. Its always below what is expected.

SH12A internal stopping powers are not very good at such low energies. We should consider ICRU / MSTAR tables instead.

These tables can be loaded with the `LOADEDX` card for mat.dat.

It requires however that we generate the stopping power tables first. This can be done with a program shield_dEdx which uses the library libdedx: https://github.com/APTG/libdedx

(It is installed on the trip computer.)

Try,
``` 
shield_dEdx 1 276
``` 
and  it will write a stopping power file for water.

``` 
shield_dEdx 1 -1
shield_dEdx 2 -1
``` 
to see a list of available materials for the various programs.

**Comment by fkalholm** [comment]
Now I get:
[Ion, MeV in article, MeV received]
12C, 33.2, 26
12C, 20.3, 13
12C, 9.1, 7
20Ne, 56.2, 52
20Ne, 34.7, 26
20Ne, 15, 7

So the stopping power tables helped somewhat, but a few items still about only 50% of expected.
@nbassler do you think it can be improved more, or is this the best estimate?

**Comment by nbassler** [comment]
> So the stopping power tables helped somewhat, but a few items still about only 50% of expected.
> @nbassler do you think it can be improved more, or is this the best estimate?

I think this is an inherent problem for this kind of experiment. The estimation of energy is very uncertain. Doubt we can make this much better as it is now, so lets leave it like that. Maybe we can ask Andrzej about the details some day.

It should go better for the Furusawa data.

File: 2008_Czub/Template/mat.dat [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+END
+
+MEDIUM 4
+*mylar
+ICRU 222
+LOADDEDX
+END
+
+MEDIUM 5
+*water
```
Comment (grzanka) [inline 2008_Czub/Template/mat.dat]: If you want be super precise, use water density 0.9982. It is different from 1 g/cm3 when water is at 20*C.

Comment (nbassler) [reply 2008_Czub/Template/mat.dat]: True.

Comment (fkalholm) [reply 2008_Czub/Template/mat.dat]: Yes, might as well be as precise as we can :)

**Comment by nbassler** [comment]
when you think it is ready to be merged, mark it as such, and request a review. :)

**Pull request merged**